language/Turkish.jsx

```diff
@@ -0,0 +1,69 @@
+    <br />
+    <p>Biz katılmıyoruz</p>
+    <hr />
+    <h2>Kalcor / SAMP  hakkında ne düşünüyorsunuz?</h2>
+    <p>Sa:MP'yi seviyoruz, bu yüzden buradayız ve bunu Kalcor'a borçluyuz. Değişiklikler için yıllar boyunca çalıştı ve bu katkılar göz ardı edilmemeli veya unutulmamalıdır. Open.mp yol açma sebebimiz, son birkaç karara itiraz ettiğimiz için yapıldı.Bu yüzden, SA: MP'yi Kalcor'suz ruhu ile sürdürmeye çalışmak için talihsiz bir karar vermeye zorlandık.Bu bizzat kendisine karşı yapılan bir eylem değildir ve buna karşı bir saldırı olarak görülmemelidir. Kişilere karşı yapılan hakaretlere kim olduklarına bakılmaksızın tahammül etmeyeceğiz. open.mp; hakkında kişilere saldırı - hakaret etmeden makul bir tartışma yapabiliriz. </p>
+    <hr />
+    <h2>Bu toplumu bölmek değil midir?</h2>
+    <p>Böyle bir niyetimiz yok. Ancak amaçımız bir kitleyi bölmek değil, her şeyin kaybolduğunu - silindiğini izlemek yerine bunu kurtarmak daha iyidir. SA:MP resmi forumundan çok sayıda insan yasaklandı (ve bazı durumlarda tüm emeklerini kaldırdılar), ancak Kalcor, resmi forumun SA:MP topluluğu değil, yalnızca bir kişi olduğunu belirtti.Birçok oyuncu ve sunucu sahibi foruma hiç katılmamış; bu yüzden bu insanlarla konuşmak toplumu daha da birleştirecek.</p>
+    <hr />
+    <h2>Bu "Open" Multiplayer olduğu için, bu proje açık kaynak kodlu olacak mı?</h2>
+    <p>Evet, planımız bu. Bu arada, geliştirme sürecinin geri bildirimi ve şeffaflığı (kendi içinde ilerleyen) açısından açıklık sağlamaya çalışıyoruz ve mümkün olduğu zaman, diğer önemli sorunların çözüldüğünde ve istikrarlı bir gelişme süreci elde edilir edilmez açık kaynak koduna paylaşmaya çalışacağız.</p>
+    <hr />
+    <h2>Open.mp ne zaman çıkacak?</h2>
+    <p>Bu eski bir sorudur ve ne yazık ki bunun için şu cevaba sahibiz; hazır olduğunda. Böyle bir projenin ne kadar süreceğini bilmek mümkün değil. </p>
+    <hr />
+    <h2>Nasıl yardımcı olabilirim?</h2>
+    <p>Foruma bir göz atın. Sadece bunun için bir konumuz var ve daha fazla yardım ihtiyaçı mevcut olduğunda bu konuyu güncel tutacağız. Projenin planlanandan biraz daha erken bir şekilde ortaya çıkmasına rağmen, şimdiden ilk lansmana gidiyoruz, ancak bu daha fazla yardımın ihtiyaçımız olmadığı anlamına gelmiyor. Projeye ilgi duyduğunuz ve inandığınız için şimdiden teşekkür ederiz:</p>
+    <br />
+    <a href="https://www.burgershot.gg/showthread.php?tid=361">
+      <u>Yardım Konusu</u>
+    </a>
+    <hr />
+    <h2>burgershot.gg nedir?</h2>
+    <p>Burgershot.gg bir oyun forumudur, başka bir şey değildir. Birçok insan dahil olmuştur ve OMP'nin bazı güncellemeleri ve gelişmeleri orada yayınlanmaktadır, ancak bunlar iki bağımsız projedir.</p>
+    <hr />
+    <h2> "OpenMP" hakkında nedir?</h2>
+    <p>
+      <a href="https://ru.wikipedia.org/wiki/OpenMP">Open Multiprocessing</a> - olarak bilinen "OpenMP", farklı projedir, bizim projemiz - "open.mp". olarak isimlendirilir. Bunlar ikisi tamamen farklı projelerdir.
```

Comment:
Should be https://tr.wikipedia.org/wiki/OpenMP?

Reply:
> Should be https://tr.wikipedia.org/wiki/OpenMP?

Thank you, edited.